improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -47,186 +47,118 @@
+                                              2.77777778e-08,
+                                              2.77777778e-07])))
+
+        data_rain = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0,
+                              0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0,
+                              0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                              0.0, 0.0, 0.0]).reshape((3, 1, 3, 3))
+        rainfall_rate = (
+            set_up_probability_threshold_cube(
+                data_rain,
+                'rainfall_rate',
+                'm s-1',
+                spp__relative_to_threshold='above',
+                forecast_thresholds=np.array([8.33333333e-09,
+                                              2.77777778e-08,
+                                              2.77777778e-07])))
+
+        data_cloud = np.array([0.0, 1.0, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 1.0,
+                               0.0, 0.0, 0.0, 0.0, 1.0, 1.0,
+                               0.0, 0.0, 1.0]).reshape((2, 1, 3, 3))
+        cloud = (set_up_probability_threshold_cube(
+            data_cloud,
+            'cloud_area_fraction',
             '1',
             spp__relative_to_threshold='above',
-            forecast_thresholds=np.array([0.85])))
-
-    data_vis = np.array([0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
-                         0.0, 0.0, 1.0, 1.0, 0.0, 0.0,
-                         0.0, 1.0, 0.0]).reshape((2, 1, 3, 3))
-    visibility = (
-        set_up_probability_threshold_cube(
-            data_vis,
-            'visibility_in_air',
-            'm',
-            spp__relative_to_threshold='below',
-            forecast_thresholds=np.array([1000.0, 5000.0])))
-    visibility.coord(var_name="threshold"
-                     ).attributes['spp__relative_to_threshold'] = 'below'
-
-    cubes = iris.cube.CubeList([snowfall_rate, rainfall_rate,
-                                cloud, cloud_low,
-                                visibility])
-    return cubes
+            forecast_thresholds=np.array([0.1875, 0.8125])))
+
+        data_cld_low = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 1.0,
+                                 0.0, 0.0, 0.0]).reshape((1, 1, 3, 3))
+        cloud_low = (
+            set_up_probability_threshold_cube(
+                data_cld_low,
+                'low_type_cloud_area_fraction',
+                '1',
+                spp__relative_to_threshold='above',
+                forecast_thresholds=np.array([0.85])))
+
+        data_vis = np.array([0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                             0.0, 0.0, 1.0, 1.0, 0.0, 0.0,
+                             0.0, 1.0, 0.0]).reshape((2, 1, 3, 3))
+        visibility = (
+            set_up_probability_threshold_cube(
+                data_vis,
+                'visibility_in_air',
+                'm',
+                spp__relative_to_threshold='below',
+                forecast_thresholds=np.array([1000.0, 5000.0])))
+
```

Comment:
Is there a reason there isn't the visibility coord anymore? (line 223 in the old code)
visibility.coord(var_name="threshold").attributes['spp_relative_to_threshold'] = 'below'

Reply:
Copy paste error. I removed the duplication of setting up the visibility cube, but kept the version without this relative to threshold attribute on the threshold coordinate. Re-added as it should be there. Good spot.